docs/reference/class/MongoDBBulkWriteResult.txt

```diff
@@ -0,0 +1,39 @@
+.. _php-bulk-write-result:
```

Comment:
Are these anchors still needed if the result classes now have their own pages? No need to clean this up now, as it can be addressed in a subsequent PR. It would entail both removing this and updating any links that refer to it (for all such redundant anchors).

Reply:
If you'd prefer I remove the content from "Enumeration Classes", "Result Classes", etc (mentioned in [this comment](https://github.com/mongodb/mongo-php-library/pull/1229/files#r1488489592)) then yes these anchors aren't necessary and I'll remove them. Otherwise, I'll keep the anchors if I make those landing pages clickable.